test/npm_nested_import_errors/main.js

```diff
@@ -0,0 +1,2 @@
+//require("dep");
```

Comment:
dep is not being imported so the error won't occur, right?

Reply:
yes, I imported here ```System.import('dep').then(null, function(err){``` line 15 of dev.html